docs/design/Chem Integration.md

```diff
@@ -0,0 +1,66 @@
+# Chem integration into Upgrade Assistant
+
+Chem is a tool developed internally to solve the problem of matching loose assemblies (ie `.dll` files added into projects with unknown provenance). As customers are looking to move to .NET 5, this may pose a challenge as these versions are highly likely to be .NET Framework compilations that may not work as expected on newer platforms. Often, these libraries have NuGet packages now and newer versions that could work on newer platforms.
```

Comment:
This is not an accurate statement. It happens to be good at this due to the data that it contains, but wasn't created for this purpose (and isn't even what it ends up being used for most). I think a public-facing statement of what Chem is for would be something like "Chem is a tool developed internally to gain insights from the ecosystem of code attached to .NET. As a result of the data that it gathers, it is able to match loose assemblies with their probable origins" (or something like that).